contents/docs/user-guides/organizations-and-projects.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Organizations and projects
+sidebar: Docs
+showTitle: true
+---
+
+For a user experience that scales for all your needs, PostHog implements the concepts of organizations and projects.
+
+## Organizations
+
+An organization is the largest building block of PostHog's structure: it's made up of projects and has PostHog users as members.
+
+Most commonly a PostHog organization represents a real-world company. But this isn't a requirement and you are free to gather people any way you see fit.
+
+As a brand new PostHog user you'll always find yourself in a single organization upon account creation.
+If you've joined from an invite, that will be the organization you've been invited to.
+Otherwise you'll become the Owner of a new organization created just for you, based on the company name you provided when creating the account.
```

Comment:
```suggestion
Otherwise, you'll become the Owner of a new organization created just for you, based on the company name you provided when creating the account.
```